pkg/fnenv/native/builtin/foreach.go

```diff
@@ -0,0 +1,82 @@
+package builtin
+
+import (
+	"fmt"
+
+	"github.com/fission/fission-workflows/pkg/types"
+	"github.com/fission/fission-workflows/pkg/types/typedvalues"
+)
+
+const (
+	Foreach            = "foreach"
+	ForeachInputHeader = "foreach"
+	ForeachInputDo     = "do"
+)
+
+// FunctionForeach is a control flow construct to execute a certain task for each element in the provided input.
+// The tasks are executed in parallel. Foreach does not gather or store the outputs of the tasks in any way.
```

Comment:
What is the recommendation to users for how to deal with task output?

Reply:
See https://github.com/fission/fission-workflows/pull/135#discussion_r182843899

Reply:
Not sure how that answers the question? If there is no output collected, what do we recommend to the users who want to process a set of data in an array?